README.md

```diff
@@ -37,6 +37,14 @@ Now visit [`http://localhost:5000/`](http://localhost:5000/) in your web browser
 ### Notes
 
 The `.env` file is used by flask to set environment variables when running `flask run`. This enables things like developement mode (which also enables features like hot reloading when you make a file change).
-* There's also a [SECRET_KEY](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY) variable which is used to encrypt the flask session cookie.
+
+Add your Trello details for the below to your `.env` file: 
+
+* KEY
+* TOKEN
```

Comment:
It looks like these are actually `TRELLO_KEY` and `TRELLO_TOKEN` respectively. It would be nice to also update `.env.template` to have the right keys by default

Reply:
Good spot, corrected. Also updated the template to reflect the required information